examples/glueviz_wcs_example.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# WCS Linking with Glue-Jupyter\n",
+    "\n",
+    "This notebook provides an example of linking two dithered NIRISS images using `glue-jupyter`. The development version of `glue-jupyter` is required. To install it:\n",
+    "\n",
+    "```\n",
+    "pip install git+https://github.com/glue-viz/glue-jupyter.git\n",
+    "```\n",
+    "\n",
+    "The versions used to develop this:\n",
+    "\n",
+    "```\n",
+    "# Name                    Version                   Build  Channel\n",
+    "glue-core                 0.15.0.dev5617           pypi_0    pypi\n",
+    "glue-jupyter              0.0.0                    pypi_0    pypi\n",
+    "glue-vispy-viewers        0.12.dev0                pypi_0    pypi\n",
```

Comment:
Note: I was also using
```
bqplot                    0.12.0a0                 pypi_0    pypi
bqplot-image-gl           0.1.4                    pypi_0    pypi
```

but I just updated `bqplot` to

```
bqplot                    0.12.0a1                 pypi_0    pypi
```

I'll have to add the version info for `bqplot*` packages next time I update this.